playground/utils/datasets.py

```diff
@@ -446,16 +446,19 @@ def load_npoints(n, noisy=False, n_variations=1):
     # else assume that it is a list or tuple of dimensionalities
     datasets = []
     for dim in n:
-        data = np.eye(dim)
-        labels = np.arange(dim)
+        leftdata = np.repeat(np.eye(dim), 4, axis=0)
```

Comment:
when naming variables or methods with compound names, please separate the words with an underscore, i.e. `left_data`, `right_data`, `private_latent` and so on.